docs/source/vertical_ref.rst

```diff
@@ -0,0 +1,6 @@
+.. _vertical_ref:
+
+Vertical referencing
+====================
+
+In construction
```

Comment:
Same comment as for the DEM. I suggest adding something easy to `grep -r` when the time comes!

Reply:
+1, we can add TODO everywhere (I didn't have that in mind and just wrote the "In construction")